build.gradle

```diff
@@ -589,7 +590,21 @@ task lnpTests(type: Test) {
     afterTest afterTestClosure
     afterSuite checkTestNgFailureClosure
     testLogging {
-        events "passed"
+        events "passed", "failed", "skipped"
+    }
+    // Display the L&P test results in the build
+    def outputCache = new LinkedList<String>()
+    beforeTest { TestDescriptor td -> outputCache.clear() } // clear output cache before the test starts
+    onOutput { TestDescriptor td, TestOutputEvent toe -> // put output in the cache
+        outputCache.add(toe.getMessage())
+        while (outputCache.size() > 2) outputCache.remove() // keep only last 2 result lines; if more than 3 lines -> drop first
```

Comment:
Not entirely sure if I follow the reasoning here? Could you help me understand this better

Reply:
This is just a precaution to prevent the build output from becoming messy. This will show the last 2 lines that are logged to the console, in the `gradlew lnpTests` build output. 
Ideally, we only want the JMeter test results to get displayed here. 

I haven't been able to extensively test this and there is an assumption made here, but this is the best workaround that I could find. The assumption is that nothing else is printed after we print the JMeter results summary.

Reply:
> if more than 3 lines

this is a typo. Should be `2` 😬